src/configure/helper/devOps/azureDevOpsHelper.ts

```diff
@@ -131,7 +131,7 @@ export class AzureDevOpsHelper {
             }
         }
 
-        if(queueId) {
+        if(queueId !== null && queueId !== undefined) {
```

Comment:
is there a difference?

Reply:
Yes. it just clicked to me. if(queueId) checks for non-null but as queueId is a number and not string, if(0) will be false when id is 0. It is non null but this condition fails for this case. Also, I will remove undefined condition as I am setting queueId to null at the start